examples/README

```diff
@@ -0,0 +1,5 @@
+The directories located here are used to differentiate between the types of notebooks that one may use for working with LSST. Short descriptions of each directory are as follows:
```

Comment:
So, if you rename this file `README.rst` GitHub will render it using rich text and they will look nicer. see https://github.com/lsst-ts/ts_notebooks/blob/tickets/DM-22689/README.rst

Also, can you rephrase this a little bit? Took me some time to figure out what it means. Maybe something like:

> This directory contains example notebooks on how to interact with the observatory control system. Notebooks are categorized and separated based on the their functionality. 
>
>Here is a short descriptions of each sub-directory:

Reply:
Completed. The README is now more comprehensive... and most importantly it looks prettier :-)